### ISSUE TITLE
Accessibility DS storybook audit

### ISSUE THREAD

**Issue by jcfilben** [grommet/hpe-design-system#4259]
Follow on to https://github.com/grommet/hpe-design-system/issues/4236

Once our DS storybook is in place we should go through and look at the accessibility tab in storybook and file issues as needed.

**Comment by jcfilben** [comment]
Storybook is available here: https://unrivaled-bublanina-3a9bae.netlify.app/
Look at the accessibility issues that are flagged in the accessibility tab of each story and file issues in the hpe-design-system-repo to document these. They should be tagged with the accessibility label and added to this board.

**Comment by luketa8** [comment]
Added Tickets:

- [4360](https://github.com/grommet/hpe-design-system/issues/4360)
- [4359](https://github.com/grommet/hpe-design-system/issues/4359)

**Comment by luketa8** [comment]
Some of these issues should be transfered to the `grommet`repo. I [recreated](https://github.com/grommet/grommet/issues/7421) issue 4362 in that board, but I don't have access to adding it to the Accessibility board or adding any labels.

@jcfilben

**Comment by jcfilben** [comment]
Thanks @luketa8! I'll take a look at the issues Tuesday and see which ones should be transferred to the grommet repo. I'll also give you access so that in the future you will be able to add labels and tickets to the board from the grommet repo
